packages/core/src/connectivity/scp-cf/connectivity-service.ts

```diff
@@ -49,9 +49,16 @@ interface HostAndPort {
   protocol: Protocol.HTTP;
 }
 
-function proxyHostAndPort(): HostAndPort {
+export function proxyHostAndPort(): HostAndPort {
   const service = readConnectivityServiceBinding();
 
+  if (typeof service.credentials.onpremise_proxy_http_port !== 'undefined') {
```

Comment:
cloudn't we just use something like:
```
port: service.credentials.onpremise_proxy_http_port || service.credentials.onpremise_proxy_port
```

Reply:
Ahh I didn't know I can use logical operators like that, that's good to know!